lib/js/components/Form/CheckboxGroupField/CheckboxGroupField.vue

```diff
@@ -0,0 +1,101 @@
+<template>
+	<form-field v-bind="formFieldProps">
+		<template #field="{ messageId, labelId }">
+			<checkbox-group-root
+				v-model="value"
+				class="ds-checkboxGroupField"
+				role="group"
+				:aria-describedby="messageId"
```

Comment:
I believe we should disable this aria when `message` is not present on a page (same with label)

Reply:
It's not trivial - I'd leave it as is or we can discuss it on the daily ;)